kano_updater/utils.py

```diff
@@ -237,12 +237,12 @@ def _handle_sigusr1(signum, frame):
 
 
 def show_relaunch_splash():
-    cmd = ["kano-updater", "ui", "relaunch-splash", str(os.getpid())]
-    p = subprocess.Popen(cmd, shell=False)
-
     # register a handler for SIGUSR1
     signal.signal(signal.SIGUSR1, _handle_sigusr1)
 
+    cmd = ["kano-updater-internal", "ui", "relaunch-splash", str(os.getpid())]
```

Comment:
Pretty sure you've checked that - but are the double dashes for these options getting magically in?

Reply:
These options don't take double dashes :-)